### PR TITLE
helloWorld commit

### PR DESCRIPTION
**Pull request by RipaAloyan**

### CODE REVIEW

File: helloWorld/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (chandranik90) [inline helloWorld/.idea/.gitignore]: You should not create .idea folder. This kind of folder is being automatically generated by IntellJ Idea when you create a project. So, it should be rather ignored. Please move your gitignore to root

File: helloWorld/helloWorld.iml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chandranik90) [inline helloWorld/helloWorld.iml]: You have pushed unnecessary iml file. This is an autogenerated file. It's project related and not part of source. So, it should be removed from version control. Please use "git rm" command in order to untrack this file and add iml extension into gitignore

File: helloWorld/.idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chandranik90) [inline helloWorld/.idea/modules.xml]: The nested .idea folder should be removed from version control at all.

**Review by chandranik90 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#ignore files with iml extenion
+*.iml
+
+
+#Ignore everything under .idea folder, not elsewhere
```
Comment (chandranik90) [inline .gitignore]: Seems, now you missed the "out" folder, which is being generated by IntelliJ as well when we run the main method.

**Review by chandranik90 — CHANGES_REQUESTED** [review]

**Comment by RipaAloyan** [comment]
Oh, that's right. Now when I get the rule(all auto-generated files most probably need to be ignored))I will add it.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

File: .gitignore (hunk 4 above)
Comment (RipaAloyan) [reply .gitignore]: I will add it now, thank you.

**Review by chandranik90 — APPROVED** [review]

**Pull request merged**